### ISSUE TITLE
Some runtime warnings

### ISSUE THREAD

**Issue by nbassler** [pytrip/pytripgui#40]
```
bassler@baslup:~$ pytripgui 
/home/bassler/.local/lib/python2.7/site-packages/pytripgui/main.py:365: wxPyDeprecationWarning: Call to deprecated item. 
  wx.InitAllImageHandlers()
/home/bassler/.local/lib/python2.7/site-packages/pytripgui/panels/plotpanel.py:351: VisibleDeprecationWarning: using a non-integer number instead of an integer will result in an error in the future
  ct_value = self.data.get_image_cube()[pos[2], pos[1], pos[0]]
```

**Comment by grzanka** [comment]
How to reproduce ?

**Comment by grzanka** [comment]
@nbassler can you check again ?

**Comment by nbassler** [comment]
PyTRiP 1.3.1
PyTRiPGUI 0.2.1

```
bassler@baslup:~/Projects/shieldhit/res/TST000$ pytripgui 

** (pytripgui:17104): WARNING **: Couldn't register with accessibility bus: Did not receive a reply. Possible causes include: the remote application did not send a reply, the message bus security policy blocked the reply, the reply timeout expired, or the network connection was broken.
/home/bassler/.local/lib/python2.7/site-packages/pytripgui/main.py:365: wxPyDeprecationWarning: Call to deprecated item. 
  wx.InitAllImageHandlers()
```

**Comment by nbassler** [comment]
/home/bassler/.local/lib/python2.7/site-packages/pytripgui/panels/plotpanel.py:351: VisibleDeprecationWarning: using a non-integer number instead of an integer will result in an error in the future
  ct_value = self.data.get_image_cube()[pos[2], pos[1], pos[0]]


that part is for sure resolved with #53 and MR #52
